src/PiWeb.Import.Sdk/Modules/ImportFormat/ImportGroupExtensions.cs

```diff
@@ -27,11 +27,12 @@ public static class ImportGroupExtensions
     /// Creates an <see cref="ImportGroup"/> that is complete and can be imported at once.
```

Comment:
The status no longer has to be complete.

Reply:
I removed the extension completely because there was not much benefit in them.